mlrun/utils/notifications/notification_pusher.py

```diff
@@ -100,8 +100,13 @@ class NotificationPusher(_NotificationPusherBase):
         "aborted": "{resource} aborted",
     }
 
-    def __init__(self, runs: typing.Union[mlrun.lists.RunList, list]):
+    def __init__(
+        self,
+        runs: typing.Union[mlrun.lists.RunList, list],
+        default_notification_params: dict = None,
+    ):
         self._runs = runs
+        self._default_notification_params = default_notification_params or {}
```

Comment:
```suggestion
        self._default_params = default_params or {}
```
I think it is redundant to name it x_notification_y as the class it notification anyway